v3/src/components/inspector-panel-wrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import { useDndContext } from "@dnd-kit/core"
+import React from "react"
+import { getTileComponentInfo } from "../models/tiles/tile-component-info"
+import { ITileModel } from "../models/tiles/tile-model"
+import { uiState } from "../models/ui-state"
+
+interface IInspectorPanelWrapper {
+  tile?: ITileModel
+}
+export function InspectorPanelWrapper({ tile }: IInspectorPanelWrapper) {
+  const { InspectorPanel } = getTileComponentInfo(tile?.content.type) || {}
+  const { active } = useDndContext()
+  const show = uiState.isFocusedTile(tile?.id) && !active
+  return InspectorPanel ? <InspectorPanel tile={tile} show={show} /> : null
```

Comment:
This hides the inspector panel during drags and only re-renders the inspector panel when its state changes rather than the component as a whole.